domainlab/models/model_diva.py

```diff
@@ -97,7 +99,11 @@ def hyper_init(self, functor_scheduler, trainer=None):
             """
             return functor_scheduler(
                 trainer=trainer,
-                beta_d=self.beta_d, beta_y=self.beta_y, beta_x=self.beta_x)
+                beta_d=self.beta_d,
+                beta_y=self.beta_y,
+                beta_x=self.beta_x,
+                gamma_d=self.gamma_d,
```

Comment:
do we have a way to judget whether the trainer is fbopt here?  For the moment, we could use this way, but the default DIVA model do not adjust gamma_d at all.